ui/src/app/pages/login/cookie-policy.tsx

```diff
@@ -0,0 +1,161 @@
+        <div style={styles.textSection}>If the settings on your software that you are using to view the <AoUTitle/> are
+          adjusted to accept cookies, we take this, and your continued use of our platform, to mean that you accept
+          this. Should you wish to not accept cookies, you can learn how to do this below.</div>
+        <SmallHeader style={styles.smallHeader}>Anonymous visitor statistics cookies</SmallHeader>
+        <div style={styles.textSection}>
+          We use cookies to compile visitor statistics including the number of visitors to our platform, technology used
+          to visit our platform (e.g. operating systems, helping to identify specific problems), time spent on the
+          platform, popular pages, etc. This helps us to continuously improve our platform. Analytics programs also tell
+          us, anonymously, how visitors found our platform (e.g. through a search engine) and whether they have been
+          here before, helping us to invest in the development of our services instead of marketing.
+        </div>
+        <div style={styles.textSection}>
+          There will also be cookies on our platform placed by third parties whose services we use to deliver our
+          services. To control third-party cookies, you can adjust your browser settings, as below.
+        </div>
+        <SmallHeader style={styles.smallHeader}>Cookies we use</SmallHeader>
+        <FlexRow>
+          <div style={{...styles.tableItem, ...styles.smallItem, ...styles.tableHeader}}>Cookie ID</div>
+          <div style={{...styles.tableItem, ...styles.smallItem, ...styles.tableHeader}}>Provider</div>
+          <div style={{...styles.tableItem, ...styles.wideItem, ...styles.tableHeader}}>Description</div>
+        </FlexRow>
+        <FlexRow>
+          <div style={{...styles.tableItem, ...styles.smallItem}}>__cfduid</div>
+          <div style={{...styles.tableItem, ...styles.smallItem}}><a style={styles.inlineAnchor}
+                                                                     href={ZENDESK_PRIVACY_LINK}>Zendesk</a></div>
+          <div style={{...styles.tableItem, ...styles.wideItem}}>Used by the CloudFlare content delivery network to
+            identify trusted web traffic from the embedded Zendesk help widget.</div>
+        </FlexRow>
+        <FlexRow>
+          <FlexColumn style={{...styles.tableItem, ...styles.smallItem}}>
+            <div>_ga*</div>
+            <div>_gid*</div>
+          </FlexColumn>
+          <div style={{...styles.tableItem, ...styles.smallItem}}><a style={styles.inlineAnchor}
+                                                                     href={GOOGLE_PRIVACY_LINK}>Google</a></div>
+          <div style={{...styles.tableItem, ...styles.wideItem}}>Used by Google Analytics to distinguish users.</div>
+        </FlexRow>
+        <FlexRow>
+          <FlexColumn style={{...styles.tableItem, ...styles.smallItem}}>
```

Comment:
I'm not really happy with this method of doing it, but semantically it feels the closest to what we could do.